dwave/system/testing.py

```diff
@@ -344,19 +370,24 @@ def sample(self, bqm, **kwargs):
         label = kwargs.get('label')
         if label is not None:
             info.update(problem_label=label)
+        
 
         #Special handling of flux_biases, for compatibility with virtual graphs
 
         flux_biases = kwargs.get('flux_biases')
         if flux_biases is not None:
             self.flux_biases_flag = True
 
-        substitute_kwargs = {'num_reads' : kwargs.get('num_reads')}
-        if substitute_kwargs['num_reads'] is None:
-            substitute_kwargs['num_reads'] = 1
+        # Create a local dictionary combining self.substitute_kwargs and relevant kwargs
```

Comment:
The substitute sampler's sampling args should go in the init.


I don't think the `sample` method should consume any `kwargs` for the substitue sampler because the mock is used to mock a `DWaveSampler`: it shouldn't consume parameters that a `DWaveSampler` doesn't.